spirv_glsl.cpp

```diff
@@ -689,22 +689,6 @@ void CompilerGLSL::emit_struct(SPIRType &type)
 		statement("");
 }
 
-Bitset CompilerGLSL::combined_decoration_for_member(const SPIRType &type, uint32_t index)
```

Comment:
moved to parent class unchanged, except declared const

Reply:
Makes sense.